src/apps/loan-list/materials/stackable-material.tsx

```diff
@@ -87,6 +87,7 @@ const StackableMaterial: React.FC<StackableMaterialProps> = ({
               </button>
             )}
           {materialIsOverdue(dueDate) && (
+            // todo 5387.2.2b.iv.3.b.i. Se Bilag 6 pkt. 2
             <a
```

Comment:
Should be Link component but I get the point that it is a TODO :)

Reply:
yep, it'll be fixed